Source/Core/ClippingPlaneCollection.js

```diff
@@ -84,9 +116,18 @@ define([
          */
         this.edgeWidth = defaultValue(options.edgeWidth, 0.0);
 
+        // If this ClippingPlaneCollection has an owner, only its owner should update or destroy it.
+        // This is because in a Cesium3DTileset multiple models may reference the tileset's ClippingPlaneCollection.
+        this._owner = undefined;
+
         this._testIntersection = undefined;
         this._unionClippingRegions = undefined;
-        this.unionClippingRegions = defaultValue(options.unionClippingRegions, false);
+        this.unionClippingRegions = defaultValue(options.unionClippingRegions, false); // set using setter
```

Comment:
I don't know that using the setter here is a good practice - it makes construction harder to follow since it is in multiple places.  _Might_ be better off duplicating that one extra line in the setter here.